<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtOpencvClass</class>
 <widget class="QWidget" name="qtOpencvClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtOpencv</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="4">
    <widget class="QLabel" name="targetPicLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QLabel" name="originPicLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="importPicBtn">
     <property name="text">
      <string>导入图片</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="createTargetBtn">
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="backgroundSelect"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="exportTargetBtn">
     <property name="text">
      <string>导图效果图</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtOpencv.qrc"/>
 </resources>
 <connections/>
</ui>
